pwiz_tools/Skyline/TestTutorial/TutorialAuditLogs/en/TestExistingExperimentsTutorial.log

```diff
@@ -17,308 +17,306 @@ Settings > Peptide Settings -- Modifications > Isotope modifications > "heavy" >
-GNVGFVFTNEPLTEIK	882.9652	700.4294	YLR340W	RPP0 SGDID:S000004332, Chr XII from 805887-806825, Verified ORF, "Conserved ribosomal protein P0 similar to rat P0, human P0, and E. coli L10e; shown to be phosphorylated on serine 302"
-GNVGFVFTNEPLTEIK	886.9722995	708.443599	YLR340W	RPP0 SGDID:S000004332, Chr XII from 805887-806825, Verified ORF, "Conserved ribosomal protein P0 similar to rat P0, human P0, and E. coli L10e; shown to be phosphorylated on serine 302"
-GNVGFVFTNEPLTEIK	882.9652	1290.7076	YLR340W	RPP0 SGDID:S000004332, Chr XII from 805887-806825, Verified ORF, "Conserved ribosomal protein P0 similar to rat P0, human P0, and E. coli L10e; shown to be phosphorylated on serine 302"
-GNVGFVFTNEPLTEIK	886.9722995	1298.721799	YLR340W	RPP0 SGDID:S000004332, Chr XII from 805887-806825, Verified ORF, "Conserved ribosomal protein P0 similar to rat P0, human P0, and E. coli L10e; shown to be phosphorylated on serine 302"
-YEIISQQPENVSNLSK	924.972	987.516	YKL035W	UGP1 SGDID:S000001518, Chr XI from 369534-371033, Verified ORF, "UDP-glucose pyrophosphorylase (UGPase), catalyses the reversible formation of UDP-Glc from glucose 1-phosphate and UTP, involved in a wide variety of metabolic pathways, expression modulated by Pho85p through Pho4p"
-YEIISQQPENVSNLSK	928.9790995	995.530199	YKL035W	UGP1 SGDID:S000001518, Chr XI from 369534-371033, Verified ORF, "UDP-glucose pyrophosphorylase (UGPase), catalyses the reversible formation of UDP-Glc from glucose 1-phosphate and UTP, involved in a wide variety of metabolic pathways, expression modulated by Pho85p through Pho4p"
-YEIISQQPENVSNLSK	924.972	1330.6727	YKL035W	UGP1 SGDID:S000001518, Chr XI from 369534-371033, Verified ORF, "UDP-glucose pyrophosphorylase (UGPase), catalyses the reversible formation of UDP-Glc from glucose 1-phosphate and UTP, involved in a wide variety of metabolic pathways, expression modulated by Pho85p through Pho4p"
-YEIISQQPENVSNLSK	928.9790995	1338.686899	YKL035W	UGP1 SGDID:S000001518, Chr XI from 369534-371033, Verified ORF, "UDP-glucose pyrophosphorylase (UGPase), catalyses the reversible formation of UDP-Glc from glucose 1-phosphate and UTP, involved in a wide variety of metabolic pathways, expression modulated by Pho85p through Pho4p"
-YEIISQQPENVSNLSK	924.972	1115.5802	YKL035W	UGP1 SGDID:S000001518, Chr XI from 369534-371033, Verified ORF, "UDP-glucose pyrophosphorylase (UGPase), catalyses the reversible formation of UDP-Glc from glucose 1-phosphate and UTP, involved in a wide variety of metabolic pathways, expression modulated by Pho85p through Pho4p"
-YEIISQQPENVSNLSK	928.9790995	1123.594399	YKL035W	UGP1 SGDID:S000001518, Chr XI from 369534-371033, Verified ORF, "UDP-glucose pyrophosphorylase (UGPase), catalyses the reversible formation of UDP-Glc from glucose 1-phosphate and UTP, involved in a wide variety of metabolic pathways, expression modulated by Pho85p through Pho4p"
-				
+Imported a transition list from the clipboard
+Extra Info: TANDVLTIR	501.778	830.474

```

Comment:
I feel like we've lost some useful information in these logs, namely the header line. At a minimum we should be including the original if there was one, and our understanding of the columns if there wasn't. Probably both would be good, really - what we saw and what we decided it meant.